nats/main.go

```diff
@@ -84,6 +85,7 @@ See 'nats cheat' for a quick cheatsheet of commands
 	ncli.Flag("domain", "JetStream domain to access").PlaceHolder("PREFIX").PlaceHolder("DOMAIN").Hidden().StringVar(&jsDomain)
 	ncli.Flag("context", "Configuration context").Envar("NATS_CONTEXT").StringVar(&cfgCtx)
 	ncli.Flag("trace", "Trace API interactions").BoolVar(&trace)
+	ncli.Flag("tls-dump-certs", "Dump TLS first certificate chain for server").BoolVar(&tlsDumpCerts)
```

Comment:
not sure this should be on the global set of flags - we already have quite a lot and I'd rather not add ones that not strictly applicable.

I can think of a few optoins here:

 * We add it to the `nats account info` which already shows a log of connection things like IPs and ports
 * We add it to context show/validate

Probably I prefer the first one

```
[rip@p1-lon]% nats account info
Connection Information:

               Client ID: 2560
               Client IP: 46.101.44.80
                     RTT: 688.062µs
       Headers Supported: true
         Maximum Payload: 1.0 MiB
       Connected Cluster: lon
           Connected URL: nats://n3-lon:4222
       Connected Address: 188.166.173.133:4222
     Connected Server ID: NBFWZTPM3AVUN4S6LONLHO4SQOMCCREHDJ5SPDM3YE3CAGZJ7KWJBC6J
   Connected Server Name: n3-lon
                     TLS: true (set --tls-detail to see certificates)

....
````

When just doing account info and its tls true add the message as on the last line if false just show `TLS: false`, and if supplied print the chain etc nicely at the bottom.